bin/icons/index.js

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
+
+const fs = require('fs').promises
+const path = require('path')
+
+const svgstore = require('svgstore')
+
+const SVGO = require('./svgo')
+const config = require('../../config')
+
+const inline = false
+
+const readConfig = () => config
+
+const resolveInput = ({ dir, use }) =>
+  use.map((name) => {
+    const file = /.svg$/.test(name) ? name : `${name}.svg`
+    return path.resolve(dir, file)
+  })
+
+const read = (files) =>
+  Promise.all(
+    files.map((file) =>
+      fs.readFile(file, 'utf8').then((data) => ({
+        data,
+        name: path.basename(file),
+        path: file,
+      }))
+    )
+  )
+
+const processSvgoOptions = (options) => {
+  const plugins = Object.entries(options.plugins).map(([name, flag]) => ({
+    [name]: flag,
+  }))
+  return { ...options, plugins }
+}
+
+const optimize = (files, options) =>
+  new Promise((resolve, reject) => {
+    const svgo = new SVGO(processSvgoOptions(options))
+    const optimized = files.map(({ data, ...meta }) => ({
+      ...meta,
+      data: svgo.optimize(data),
+    }))
+    Promise.all(optimized).then(resolve, reject)
+  })
+
+const combine = (files, { name = 'sprite.svg', prefix = '' } = {}) => {
+  const data = files
+    .reduce((sprites, file) => {
+      const id = `${prefix}${path.basename(file.name, '.svg')}`
+      return sprites.add(id, file.data)
+    }, svgstore({ inline }))
+    .toString()
+  return { path: name, data }
+}
+
+const sourceOptions = {
+  plugins: {
+    removeViewBox: false,
+    removeDimensions: true,
+  },
+}
+
+const spriteOptions = {
+  plugins: {
+    removeUselessDefs: false,
+    cleanupIDs: false,
+  },
+}
+
+const run = async () => {
+  const config = await readConfig()
+  const paths = await resolveInput(config.input)
+  const files = await read(paths)
+  const icons = await optimize(files, sourceOptions)
```

Comment:
For some reason, SVGO removes all content leaving only `<svg>` tag. I have no idea what the problem may happen.